CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # CHANGELOG
 
+## v14.1.0.0
+
+* Upgrade to [Cumulus v14.1.0](https://github.com/nasa/Cumulus/releases/tag/v14.1.0)
+* Exposes the new `cloudwatch_log_retention_periods` as mentioned in the release
+notes in case a DAAC wants to modify the retention of any CloudWatch log groups.
+* updated the terraform `aws` provider in the `cumulus` and `data-persistence` modules
+to match those in the underlying Cumulus modules.
+* **Reminder** - this version requires
+[Cumulus Dashboard v12.0.0](https://github.com/nasa/cumulus-dashboard/releases/tag/v12.0.0)
+* Also, any ECS tasks are required to use the `cumuluss/cumulus-ecs-task:1.8.0`
+docker image.  This requirement is listed in the
+[Cumulus v11.1.8](https://github.com/nasa/Cumulus/releases/tag/v14.1.0)
```

Comment:
Link name doesn't match
```suggestion
[Cumulus v14.1.0](https://github.com/nasa/Cumulus/releases/tag/v14.1.0)
```

Reply:
The ECS image breaking change is only listed in the v11.1.8 release notes, I sent them a slack asking that they add it to the other releases.  I will update the CHANGELOG to change the url to point to the 1.1.8 release.